app/models/bg.rb

```diff
@@ -0,0 +1,3 @@
+class Bg < ActiveRecord::Base
```

Comment:
Bg ???????????????????????
i dont understand Modal
if it is a blog controller name should be Blog or Blogger
dont use shortcut for Class names, method names, variable name etc
kindly strictly follow this naming convention